packages/cli/yaml/yaml-schema/src/schemas/PaginationSchema.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+export const PaginationSchema = z.object({
+    page: z.string(),
+    next: z.string(),
+    results: z.string()
+});
```

Comment:
Thoughts on start to using the [.describe()](https://zod.dev/?id=describe) method on these parameters -- would love to eventually start adding auto-generated reference documentation and I think this will help more folks in the codebase